content/participants/ttjc@Goutam192002/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Goutam
+type: "participant"
+name: "Goutam B Seervi"
+portfolio: "https://goutambseervi.me"
+github-link: "https://github.com/Goutam192002"
+linkedin-link: "https://linkedin.com/in/goutambseervi"
+looking-for: "job"
+description: "Full Stack Web, Android and DevOps Enthusiast."
+---
+
+Hi! I am a Goutam B Seervi let me walk you through my intro, projects and blogs.
```

Comment:
Some typos "a Goutam" ???

Reply:
Yeah 😅 ..thanks for pointing that out. Fixing it.